README.md

```diff
@@ -1,56 +1,59 @@
 killbill-client-php
 ===================
 
-PHP client library for killbill
+PHP client library for [Killbill](http://killbill.io)
 
 Configuration
 -------------
 
 In order to user the library, you need to:
 
-1. Download the library and unzip it in your application
-2. Require the `lib/killbill.php` script
-3. Point the library to your Killbill instance via the `Killbill_Client::$serverUrl` variable (http://127.0.0.1:8080 by default)
+1. Require the library via [composer](https://getcomposer.org): `composer require killbill\killbill-client`
```

Comment:
Is there any new requirement regarding the minimal PHP version supported by the library? If so, we should probably add it in the README.